spec/unit/mutant/registry_spec.rb

```diff
@@ -0,0 +1,47 @@
+RSpec.describe Mutant::Registry do
+  let(:lookup)             { object.lookup(type)           }
+  let(:object)             { described_class.new           }
+  let(:mutator)            { class_double(Mutant::Mutator) }
+  let(:node)               { s(:true)                      }
+  let(:expected_arguments) { [node, nil]                   }
+
+  def register_mutator
+    object.register(type, mutator)
+  end
+
+  context 'on registered type' do
+    subject { register_mutator }
+
+    let(:type) { :true }
+
+    before { subject }
+
+    it 'returns registered mutator' do
+      expect(lookup).to be(mutator)
+    end
+
+    it_behaves_like 'a command method'
+
+    context 'when registered twice' do
+      it 'fails upon registration' do
+        expect { register_mutator }.to raise_error(described_class::RegistryError, 'Duplicate type registration: :true')
```

Comment:
I know that rubocop on mutant right now does not enforce it, but could we change these overlong (and the similar ones below) lines to conform to 79 chars?